server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -202,7 +203,7 @@ while read size result ;do
         continue
     fi
 
```

Comment:
Comment on line 196 is misleading: please fix it. Also `$incoming` should not exist as a directory at all (see my previous comment). Check if it *does* exist: if so, we count it as an error and continue.